src/spaces/rec2100-pq.js

```diff
@@ -12,6 +12,7 @@ const c3 = 2392 / (2 ** 7);
 
 export default new RGBColorSpace({
 	id: "rec2100pq",
+	cssId: "--rec2100-pq",
```

Comment:
This should be undashed https://drafts.csswg.org/css-color-hdr/#valdef-color-rec2100-pq